modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/RepairSign.java

```diff
@@ -122,7 +122,7 @@ private void signRightClick(PlayerInteractEvent event){
             event.getPlayer().sendMessage(I18nSupport.getInternationalisedString("Repair functionality is disabled or WorldEdit was not detected"));
             return;
         }
-        String repairName = event.getPlayer().getName();
+        String repairName = event.getPlayer().getUniqueId().toString();
```

Comment:
Since repairs are saved in folders, there is no need to also attach the UUID.

Reply:
That is for getting values such as number of different/missing blocks, missing block positions and materials that are missing

Reply:
Ah, should probably rename it to clarify that then.